### PR TITLE
Revert "fix broken links to conda-forge docs"

### PR DESCRIPTION
**Pull request by scopatz**
Reverts conda-forge/conda-smithy#1041

### CODE REVIEW

File: conda_smithy/feedstock_content/.github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Checklist
 * [ ] Used a fork of the feedstock to propose changes
 * [ ] Bumped the build number (if the version is unchanged)
 * [ ] Reset the build number to `0` (if the version changed)
-* [ ] [Re-rendered]( https://conda-forge.readthedocs.io/en/latest/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
+* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
```
Comment (isuruf) [inline conda_smithy/feedstock_content/.github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
* [ ] [Re-rendered]( http://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
```

**Comment by scopatz** [comment]
Thanks @isuruf!

File: conda_smithy/lint_recipe.py [hunk 2]
```diff
@@ -300,7 +300,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
         lints.append(
             "Using pinned numpy packages is a deprecated pattern.  Consider "
             "using the method outlined "
-            "[here](https://conda-forge.readthedocs.io/en/latest/meta.html#building-against-numpy)."
+            "[here](https://conda-forge.org/docs/meta.html#building-against-numpy)."
```
Comment (isuruf) [inline conda_smithy/lint_recipe.py]: This is still broken, but couldn't find a replacement.

Comment (scopatz) [reply conda_smithy/lint_recipe.py]: This is the new link.  I'll change it: http://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy

File: conda_smithy/lint_recipe.py [hunk 3]
```diff
@@ -392,7 +392,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
         lints.append(
             "Using toolchain directly in this manner is deprecated.  Consider "
             "using the compilers outlined "
-            "[here](https://conda-forge.readthedocs.io/en/latest/meta.html#compilers)."
+            "[here](https://conda-forge.org/docs/meta.html#compilers)."
```
Comment (isuruf) [inline conda_smithy/lint_recipe.py]:
```suggestion
            "[here](http://conda-forge.org/docs/maintainer/knowledge_base.html#compilers)."
```

File: conda_smithy/lint_recipe.py [hunk 4]
```diff
@@ -405,7 +405,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
             if "python setup.py install" in script:
                 hints.append(
                     "Whenever possible python packages should use pip. "
-                    "See https://conda-forge.readthedocs.io/en/latest/meta.html#use-pip"
+                    "See https://conda-forge.org/docs/meta.html#use-pip"
```
Comment (isuruf) [inline conda_smithy/lint_recipe.py]:
```suggestion
                    "See http://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip"
```

File: conda_smithy/lint_recipe.py [hunk 5]
```diff
@@ -433,7 +433,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
             if no_arch_possible:
                 hints.append(
                     "Whenever possible python packages should use noarch. "
-                    "See https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages"
+                    "See https://conda-forge.org/docs/meta.html#building-noarch-packages"
```
Comment (isuruf) [inline conda_smithy/lint_recipe.py]:
```suggestion
                    "See http://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds"
```

File: conda_smithy/update_cb3.py [hunk 6]
```diff
@@ -373,7 +373,7 @@ def update_cb3(recipe_path, conda_build_config_path):
                         msg = (
                             "Removing pinnings for {} to use values from "
                             "conda_build_config.yaml. If you need the pin see "
-                            "[here](https://conda-forge.readthedocs.io/en/latest/meta.html#pinning-packages) "
+                            "[here](https://conda-forge.org/docs/meta.html#pinning-packages) "
```
Comment (isuruf) [inline conda_smithy/update_cb3.py]:
```suggestion
                            "[here](http://conda-forge.org/docs/maintainer/pinning_deps.html) "
```

File: conda_smithy/update_cb3.py [hunk 7]
```diff
@@ -384,7 +384,7 @@ def update_cb3(recipe_path, conda_build_config_path):
                             msg = (
                                 "Pinning numpy using pin_compatible. If you need to pin numpy "
                                 "to a specific version see "
-                                "[here](https://conda-forge.readthedocs.io/en/latest/meta.html"
+                                "[here](https://conda-forge.org/docs/meta.html"
                                 "#building-against-numpy)."
```
Comment (isuruf) [inline conda_smithy/update_cb3.py]: http://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy

**Comment by ltalirz** [comment]
Thanks, and sorry for causing confusion.

I checked that all the new links work.
One minor comment: Many of them are using http while some are using https. Up to you whether you want to homogenize this.

**Comment by scopatz** [comment]
Please feel free to suggest changes to fix this. Thanks @ltalirz

File: conda_smithy/feedstock_content/.github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -6,7 +6,7 @@ Checklist
 * [ ] Used a fork of the feedstock to propose changes
 * [ ] Bumped the build number (if the version is unchanged)
 * [ ] Reset the build number to `0` (if the version changed)
-* [ ] [Re-rendered]( https://conda-forge.readthedocs.io/en/latest/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
+* [ ] [Re-rendered]( http://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
```
Comment (ltalirz) [inline conda_smithy/feedstock_content/.github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
```

File: conda_smithy/lint_recipe.py [hunk 9]
```diff
@@ -300,7 +300,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
         lints.append(
             "Using pinned numpy packages is a deprecated pattern.  Consider "
             "using the method outlined "
-            "[here](https://conda-forge.readthedocs.io/en/latest/meta.html#building-against-numpy)."
+            "[here](http://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy)."
```
Comment (ltalirz) [inline conda_smithy/lint_recipe.py]:
```suggestion
            "[here](https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy)."
```

File: conda_smithy/lint_recipe.py [hunk 10]
```diff
@@ -392,7 +392,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
         lints.append(
             "Using toolchain directly in this manner is deprecated.  Consider "
             "using the compilers outlined "
-            "[here](https://conda-forge.readthedocs.io/en/latest/meta.html#compilers)."
+            "[here](http://conda-forge.org/docs/maintainer/knowledge_base.html#compilers)."
```
Comment (ltalirz) [inline conda_smithy/lint_recipe.py]:
```suggestion
            "[here](https://conda-forge.org/docs/maintainer/knowledge_base.html#compilers)."
```

File: conda_smithy/lint_recipe.py [hunk 11]
```diff
@@ -405,7 +405,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
             if "python setup.py install" in script:
                 hints.append(
                     "Whenever possible python packages should use pip. "
-                    "See https://conda-forge.readthedocs.io/en/latest/meta.html#use-pip"
+                    "See http://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip"
```
Comment (ltalirz) [inline conda_smithy/lint_recipe.py]:
```suggestion
                    "See https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip"
```

File: conda_smithy/lint_recipe.py [hunk 12]
```diff
@@ -433,7 +433,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
             if no_arch_possible:
                 hints.append(
                     "Whenever possible python packages should use noarch. "
-                    "See https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages"
+                    "See http://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds"
```
Comment (ltalirz) [inline conda_smithy/lint_recipe.py]:
```suggestion
                    "See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds"
```

File: conda_smithy/update_cb3.py [hunk 13]
```diff
@@ -373,7 +373,7 @@ def update_cb3(recipe_path, conda_build_config_path):
                         msg = (
                             "Removing pinnings for {} to use values from "
                             "conda_build_config.yaml. If you need the pin see "
-                            "[here](https://conda-forge.readthedocs.io/en/latest/meta.html#pinning-packages) "
+                            "[here](http://conda-forge.org/docs/maintainer/pinning_deps.html) "
```
Comment (ltalirz) [inline conda_smithy/update_cb3.py]:
```suggestion
                            "[here](https://conda-forge.org/docs/maintainer/pinning_deps.html) "
```

File: conda_smithy/update_cb3.py [hunk 14]
```diff
@@ -384,8 +384,8 @@ def update_cb3(recipe_path, conda_build_config_path):
                             msg = (
                                 "Pinning numpy using pin_compatible. If you need to pin numpy "
                                 "to a specific version see "
-                                "[here](https://conda-forge.readthedocs.io/en/latest/meta.html"
-                                "#building-against-numpy)."
+                                "[here](http://conda-forge.org/docs/maintainer/"
```
Comment (ltalirz) [inline conda_smithy/update_cb3.py]:
```suggestion
                                "[here](https://conda-forge.org/docs/maintainer/"
```

**Comment by scopatz** [comment]
Thanks a ton!

**Comment by CJ-Wright** [comment]
LGTM thank you very much everyone!

**Pull request merged**